samples/02_power_users_developers/introduction_to_hub_initiatives_and_sites.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>Linking a page to a site is a two step process. First, you will fetch the page of interest, then you will link it to the site object with a slug for that page.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>